projects/storefrontlib/src/cms-components/cart/cart-shared/cart-item-list/cart-item-list.component.ts

```diff
@@ -89,6 +95,7 @@ export class CartItemListComponent {
   constructor(
     protected activeCartService: ActiveCartService,
     protected selectiveCartService: SelectiveCartService,
+    protected userIdService: UserIdService,
```

Comment:
In this case `userIdService` should also be optional here like `featureConfigService` and `multiCartService`

Reply:
fixed + remove unused import in the saved cart list for the userIdService